### PR TITLE
Error in console after publishing a shortcut with an image in target #260

### PR DESCRIPTION
**Pull request by edloidas**
Disabled `liveEditModel` update for content without the page editor.

### CODE REVIEW

File: src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts [hunk 1]
```diff
@@ -1246,17 +1246,20 @@ export class ContentWizardPanel
...
-                    const needsReload = !this.isSaving(); // pageModel is updated so we need reload unless we're saving already
-                    if (livePanel) {
-                        livePanel.setModel(this.liveEditModel);
-                        if (needsReload && reloadPage) {
-                            this.debouncedEditorRefresh(true);
-                        }
-                    }
-                    return needsReload;
-                });
+                const livePanel = this.getLivePanel();
```
Comment (alansemenov) [inline src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts]: Why did you move this line out of the `then` handler? It won't be used if `this.liveEditModel` is undefined, so I don't see why we need it here.

Comment (edloidas) [reply src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts]: You are right. Moved it inside the `then` scope.
